### PR TITLE
adding colorspacious and viscm recipes

### PR DESCRIPTION
**Pull request by kthyng**

### CODE REVIEW

**Comment by kthyng** [comment]
Would you like to be added as maintainers @njsmith and @stefanv?

**Comment by njsmith** [comment]
Thanks for the offer (and for working on this in general!), but I don't really have the time to figure out how to be a conda-forge maintainer right now.

**Comment by stefanv** [comment]
Thanks, @kthyng: I don't use conda myself, but I'm happily cheering from the side!

File: recipes/colorspacious/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  imports:
+    - colorspacious
+
+about:
+  home: https://github.com/njsmith/colorspacious
+  license: MIT
+  license_family: MIT
+  license_file: ''
+  summary: A powerful, accurate, and easy-to-use Python library for doing colorspace conversions
+  description: "colorspacious\n=============\n\n.. image:: https://travis-ci.org/njsmith/colorspacious.svg?branch=master\n   :target: https://travis-ci.org/njsmith/colorspacious\n   :alt: Automated test\
```
Comment (ocefpaf) [inline recipes/colorspacious/meta.yaml]: @kthyng we usually remove some of the boilerplate that conda-skeleton creates to get a cleaner recipe. Let me push a commit here to show you.

**Comment by ocefpaf** [comment]
My guess is that both recipes can be `noarch`, let's try.

**Comment by msarahan** [comment]
Looks like colorspacious needs numpy to be present at install time: https://travis-ci.org/conda-forge/staged-recipes/builds/404145336#L350

**Comment by ocefpaf** [comment]
Same thing for `viscm` :unamused:

**Comment by ocefpaf** [comment]
Thanks @kthyng!

**Pull request merged**